engine/execution/state/state.go

```diff
@@ -84,6 +84,9 @@ type ExecutionState interface {
 		ctx context.Context,
 		result *execution.ComputationResult,
 	) error
+
+	// only available with storehouse enabled
+	GetHighestFinalizedExecuted() uint64
```

Comment:
What should happen if it's called when storehouse is dissabled? maybe explain in comment?